ignite/cmd/generate_typescript_client.go

```diff
@@ -0,0 +1,39 @@
+package ignitecmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ignite-hq/cli/ignite/pkg/clispinner"
```

Comment:
This package doesn't exist anymore so the spinner related code must be refactored to use the `ignite/pkg/cliui` package.

Reply:
```suggestion
	"github.com/ignite/cli/ignite/pkg/cliui"
```